Lib/idlelib/help.html

```diff
@@ -636,19 +636,8 @@
 <h3>25.5.4.3. Extensions<a class="headerlink" href="#extensions" title="Permalink to this headline">¶</a></h3>
 <p>IDLE contains an extension facility.  Preferences for extensions can be
 changed with Configure Extensions. See the beginning of config-extensions.def
-in the idlelib directory for further information.  The default extensions
-are currently:</p>
-<ul class="simple">
-<li>FormatParagraph</li>
-<li>AutoExpand</li>
-<li>ZoomHeight</li>
-<li>ScriptBinding</li>
-<li>CallTips</li>
-<li>ParenMatch</li>
-<li>AutoComplete</li>
-<li>CodeContext</li>
-<li>RstripExtension</li>
-</ul>
+in the idlelib directory for further information.  There are currently no default extensions.</p>
+
```

Comment:
help.html is generated from Doc/library/idle.rst.  So replace this edit with an edit of the latter.  I believe it should be straightforward.